client/webserver/locales/en-us.go

```diff
@@ -466,35 +466,36 @@ var EnUS = map[string]string{
-	"Total rewards":               "Total rewards",
-	"Votes cast":                  "Votes cast",
-	"VSP":                         "VSP",
-	"select_vsp_from_list":        "Select a Voting Service Provider",
-	"Or add custom VSP URL:":      "Or add custom VSP URL:",
-	"Custom VSP URL":              "Custom VSP URL",
-	"Set Votes":                   "Set Votes",
-	"agendas":                     "agendas",
-	"Agendas":                     "Agendas",
-	"treasury spends":             "treasury spends",
-	"staking_unavailable_for_rpc": "Staking unavailable for RPC wallets using SPV",
-	"Purchase Tickets":            "Purchase Tickets",
-	"Tickets":                     "Tickets",
-	"Ticket Price":                "Ticket Price",
-	"Vote Reward":                 "Vote Reward",
-	"fee_rate_percent":            "Fee Rate (%)",
-	"Live Tickets":                "Live Tickets",
-	"Current Price":               "Current Price",
-	"Available Balance":           "Available Balance",
-	"ticket_qty_label":            "How Many?",
-	"Ticket History":              "Ticket History",
-	"No tickets to show":          "No tickets to show",
-	"Treasury Spends":             "Treasury Spends",
-	"No":                          "No",
-	"Yes":                         "Yes",
-	"Treasury Keys":               "Treasury Keys",
+	"Price":                        "Price",
+	"Ticket":                       "Ticket",
+	"Staking":                      "Staking",
+	"Active tickets":               "Active tickets",
+	"Tickets bought":               "Tickets bought",
+	"Total rewards":                "Total rewards",
+	"Votes cast":                   "Votes cast",
+	"VSP":                          "VSP",
+	"select_vsp_from_list":         "Select a Voting Service Provider",
+	"Or add custom VSP URL:":       "Or add custom VSP URL:",
+	"Custom VSP URL":               "Custom VSP URL",
+	"Set Votes":                    "Set Votes",
+	"agendas":                      "agendas",
+	"Agendas":                      "Agendas",
+	"treasury spends":              "treasury spends",
+	"staking_unavailable_for_rpc":  "Staking unavailable for RPC wallets using SPV",
+	"Purchase Tickets":             "Purchase Tickets",
+	"Tickets":                      "Tickets",
+	"Ticket Price":                 "Ticket Price",
+	"Vote Reward":                  "Vote Reward",
+	"fee_rate_percent":             "Fee Rate (%)",
+	"Live Tickets":                 "Live Tickets",
+	"Current Price":                "Current Price",
+	"Available Balance":            "Available Balance",
+	"ticket_qty_label":             "How Many?",
+	"Ticket History":               "Ticket History",
+	"No tickets to show":           "No tickets to show",
+	"Treasury Spends":              "Treasury Spends",
+	"No":                           "No",
+	"Yes":                          "Yes",
+	"Treasury Keys":                "Treasury Keys",
+	"expired_bonds_pending_refund": "Expired Bonds Pending Refund",
```

Comment:
Maybe just "Bonds Pending Refund" for the value?

Reply:
Okay.